test-apps/display-performance-test-app/src/backend/backend.ts

```diff
@@ -36,28 +37,67 @@ export async function initializeBackend() {
   const iModelHost = new IModelHostConfiguration();
   const iModelClient = new IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX ?? ""}api.bentley.com/imodels`}});
   iModelHost.hubAccess = new BackendIModelsAccess(iModelClient);
+  iModelHost.cacheDir = process.env.BRIEFCASE_CACHE_LOCATION;
```

Comment:
@gdatbentley 5 lines up is this NODE_TLS_REJECT_UNAUTHORIZED needed? It's been there since dpta was born. @kaung001 was asking if we can remove it to address potential code vulnerability.

Reply:
I'll try removing this, thanks